drivers/mtd_sdcard/mtd_sdcard.c

```diff
@@ -25,7 +25,7 @@
 #include "sdcard_spi.h"
 #include "sdcard_spi_internal.h"
 
-#include <stdint.h>
+#include <inttypes.h>
```

Comment:
`inttypes.h` [includes `<stdint.h>`](http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/inttypes.h.html#tag_13_20) that's why I removed it.